index.html

```diff
@@ -781,6 +781,17 @@ <h3>The <dfn><code>gotpointercapture</code> event</dfn></h3>
                 <h3>The <dfn><code>lostpointercapture</code> event</dfn></h3>
                 <p>A user agent MUST <a>fire a pointer event</a> named <code>lostpointercapture</code> after pointer capture is released for a pointer. This event MUST be fired prior to any subsequent events for the pointer after capture was released. This event is fired at the element from which pointer capture was removed. Subsequent events for the pointer follow normal hit testing mechanisms (out of scope for this specification) for determining the event target. See the <a href="#releasing-pointer-capture">Releasing Pointer Capture</a>, <a href="#implicit-release-of-pointer-capture">Implicit Release of Pointer Capture</a>, and <a href="#process-pending-pointer-capture">Process Pending Pointer Capture</a> sections.</p>
             </section>
+            <section>
+                <h3>The <dfn><code>click</code>, <code>auxclick</code>, and <code>contextmenu</code> events</dfn></h3>
+                <p>This section is an addition to <a href="https://w3c.github.io/uievents/#event-type-click">click</a> and
+                   <a href="https://w3c.github.io/uievents/#event-type-auxclick">auxclick</a> events defined in [[UIEVENTS]] and
+                   <a href="https://html.spec.whatwg.org/multipage/indices.html#event-contextmenu">contextmenu</a> defined in [[HTML]].
+                   The type of these events MUST be <code>PointerEvent</code> but the dispatch process is going to match that of the original specification.
+                   For these events, all <code>PointerEvent</code> specific attributes (defined in this spec) other than <code>pointerId</code> and <code>pointertype</code> will have their default values.
+                   <code>pointerId</code> and <code>pointertType</code> of these events MUST be the same as the PointerEvents that caused these events if they are generated by a PointerEvent stream.
```

Comment:
One more "stream" to get rid of.  Here is a suggestion:

> The <code>pointerId</code> and <code>pointertType</code> attributes of these events MUST be the same as the PointerEvents that caused these events if they are generated by a pointing device. 
 If the events are generated by a non-pointing device (such as voice recognition software or a keyboard interaction), <code>pointerId</code> MUST be 0 and <code>pointerType</code> MUST be an empty string.

Reply:
Done